pkg/util/ipaddr/ip.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
probably need to silence the linter on the copyright thing
make a comment that this is a copy from the v1.16 code & why this is needed

Reply:
Done